nextflow_schema.json

```diff
@@ -168,7 +168,7 @@
             "fa_icon": "fas fa-barcode"
         },
         "presto_options": {
-            "title": "Presto options",
+            "title": "pResto options",
```

Comment:
If you want to maintain Immcantation's style: pRESTO (https://presto.readthedocs.io/en/stable/)

Reply:
right, thanks!